tests/jax/systems/systems_test_data.py

```diff
@@ -87,6 +90,7 @@ def network_factory(*args: Any, **kwargs: Any) -> Any:
         nodes_on_gpu=[],
         sequence_length=4,
         period=4,
+        checkpoint_minute_interval=1 / 60,
```

Comment:
similarly, please update to be 1 instead of 1 / 60 and update test accordingly

Reply:
I set this to 1/60 meaning 1 second, perhaps 5 seconds is better. In the actual run, we want to checkpoint over many minutes

Reply:
I see!